
const deleteUsers = async () => {
    try {
        
    } catch (error) {
        
    }
}
module.exports = {deleteUsers}